src/ui/features/onboarding/Success/ReferralCodeWidget/ReferralCodeWidget.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState } from 'react';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import { Button } from 'src/ui/ui-kit/Button';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import GiftIcon from 'jsx:src/ui/assets/gift.svg';
+import ClearSolidIcon from 'jsx:src/ui/assets/clear-solid.svg';
+import CheckIcon from 'jsx:src/ui/assets/check.svg';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { collectData } from 'src/ui/shared/form-data';
+import { useMutation } from '@tanstack/react-query';
+import { walletPort } from 'src/ui/shared/channels';
+import { Input } from 'src/ui/ui-kit/Input';
+import { Spacer } from 'src/ui/ui-kit/Spacer';
+import { ZStack } from 'src/ui/ui-kit/ZStack';
+import type { ReferrerData } from 'src/modules/zerion-api/requests/check-referral';
+import * as styles from './styles.module.css';
+
+function GradientText({ children }: { children: React.ReactNode }) {
+  return <div className={styles.gradientText}>{children}</div>;
+}
+
+function EnterReferralCodeButton({ onClick }: { onClick: () => void }) {
+  return (
+    <>
+      <Spacer height={70} />
+      <VStack gap={8} style={{ justifyContent: 'center' }}>
+        <Button
+          kind="regular"
+          style={{
+            padding: '10px 16px',
+            ['--button-background' as string]: 'var(--always-white)',
+            ['--button-background-hover' as string]: '#f0f0f2',
+          }}
+          onClick={onClick}
+        >
+          <HStack gap={8}>
+            <GiftIcon />
+            <GradientText>
+              <UIText kind="body/accent">Enter Referral Code</UIText>
+            </GradientText>
```

Comment:
Why not like this?
```suggestion
            <UIText kind="body/accent" className={styles.gradientText}>Enter Referral Code</UIText>
```

Reply:
Thanks! I thought I might reuse it at some point, but that never happened. Updated ✅